importers/algod/algod_importer_test.go

```diff
@@ -0,0 +1,46 @@
+package algodimporter
+
+import (
+	"testing"
+
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+var lg *logrus.Logger
+
+func TestMetadata(t *testing.T) {
+	bot := New()
+	metadata := bot.Metadata()
+	assert.Equal(t, metadata.Name(), "algod")
+	assert.Equal(t, metadata.Description(), "Importer for fetching block from algod rest endpoint.")
+	assert.Equal(t, metadata.Deprecated(), false)
+}
+
+func TestClose(t *testing.T) {
+	bot := New()
+	bot.Close()
+	assert.Equal(t, bot, &algodImporter{})
+}
+
+func TestInit(t *testing.T) {
+	bot := New()
+	s := "netaddr: ''\ntoken: ''"
+	err := bot.Init(s, lg)
```

Comment:
does this work? isn't lg a nil pointer?

Reply:
it works, but ill update test to initialize it.